bundles/org.eclipse.search/search/org/eclipse/search/internal/ui/text/FileSearchPage.java

```diff
@@ -276,6 +276,7 @@ private IFile mostNestedEquivalent(IFile resource) {
 		ITextFileBuffer textFileBuffer = textFileBufferManager.getTextFileBuffer(resource.getFullPath(),
 				LocationKind.IFILE);
 		return Arrays.stream(resource.getWorkspace().getRoot().findFilesForLocationURI(resource.getLocationURI())) //
+		        .filter(aFile -> aFile.getProject().isOpen()) // Check File is in an Open project
```

Comment:
Although not sure if it will fix it this way
isAccessible == exists, not sure if in a closed project this would always be false.
`Returns whether this resource is accessible. For files and folders, this is equivalent to existing; for projects, this is equivalent to existing and being open. The workspace root is always accessible.`

Reply:
I think you're right, so let's stick to your proposal. Thanks for checking anyway!